docs/PROMPT_TEMPLATE_LANGUAGE.md

```diff
@@ -2,4 +2,4 @@
 
 This document has been moved to the Semantic Kernel Documentation site. You can find it by navigating to the [Prompt template syntax](https://learn.microsoft.com/en-us/semantic-kernel/prompt-engineering/prompt-template-syntax) page.
 
-To make an update on the page, file a PR on the [docs repo.](https://github.com/MicrosoftDocs/semantic-kernel-docs/blob/main/semantic-kernel/prompt-engineering/prompt-template-syntax.md)
+To make an update on the page, file a PR on the [docs repo.](https://github.com/MicrosoftDocs/semantic-kernel-docs/blob/main/semantic-kernel/prompts/prompt-template-syntax.md)
```

Comment:
Is this still a valid link? I am surprised our link check action doesn't catch this.

Reply:
Seems this `PROMPT_TEMPLATE_LANGUAGE.md` should go away since none of these links resolve. Can't find a specific prompt template syntax docs page either.